ethereum/erc20/tokens.sql

```diff
@@ -5404,6 +5404,145 @@ COPY erc20.tokens (contract_address, symbol, decimals) FROM stdin;
+\\xd34a24006b862f4e9936c506691539d6433ad297	AQUA	18
+\\xe6f143a0e0a8f24f6294ce3432ea10fad0206920	PKEX	18
+\\xeb61393503691c6e20eafd82655b7a94450c3124	LADY	18
+\\xfb130d93e49dca13264344966a611dc79a456bc5	DOGEGF	18
+\\xfb782396c9b20e564a64896181c7ac8d8979d5f4	DIVER	18
+\\x025c9f1146d4d94f8f369b9d98104300a3c8ca23	MTGY	18
+\\x120a3879da835a5af037bb2d1456bebd6b54d4ba	RVST	18
+\\x1d7ca62f6af49ec66f6680b8606e634e55ef22c1	START	18
+\\x73d7c860998ca3c01ce8c808f5577d94d545d1b4	IXS	18
+\\x8861cff2366c1128fd699b68304ad99a0764ef9a	CYC	18
+\\x9783b81438c24848f85848f8df31845097341771	COLLAR	18
+\\xd7f0cc50ad69408ae58be033f4f85d2367c2e468	VERA	18
+\\xd932ba8adf16009e5571648e070f9eebb2f259c0	AES	18
+\\xef53462838000184f35f7d991452e5f25110b207	KFT	18
+\\xfc05205ff402849b081e72ae4f8fc93cbdf40db1	Lelouch	9
+\\x058bc8ef040bd3971418e36aa88b64899378ccf4	DONA	18
+\\x07a0ad7a9dfc3854466f8f29a173bf04bba5686e	SLM	18
+\\x232afce9f1b3aae7cb408e482e847250843db931	SHARK	18
+\\x560cc7de81b2a594f6518713cbe122bcf297a6e8	i7	18
+\\x60e46a4dd91d10506d8efa2caa266e7191fe7ea8	FEWGO	18
+\\x6b4d5e9ec2acea23d4110f4803da99e25443c5df	aKLIMA	18
+\\x77dce26c03a9b833fc2d7c31c22da4f42e9d9582	DVD	18
+\\x9c4a4204b79dd291d6b6571c5be8bbcd0622f050	TCR	18
+\\xdc59ac4fefa32293a95889dc396682858d52e5db	BEAN	6
+\\xe7f4c89032a2488d327323548ab0459676269331	WAIFU	18
+\\xfa898efdb91e35bd311c45b9b955f742b6719aa2	APED	18
+\\xd502f487e1841fdc805130e13eae80c61186bc98	ITGR	18
+\\x09ce2b746c32528b7d864a1e3979bd97d2f095ab	DFL	18
+\\x4ec1b60b96193a64acae44778e51f7bff2007831	EDGE	18
+\\x6a55b1105497ce7939349f83cfe8b966845b1eb8	DGLD	18
+\\x948c70dc6169bfb10028fdbe96cbc72e9562b2ac	XP	18
+\\x9e10f61749c4952c320412a6b26901605ff6da1d	THEOS	18
+\\xa6f7645ed967faf708a614a2fca8d4790138586f	MNY	18
+\\xe7f72bc0252ca7b16dbb72eeee1afcdb2429f2dd	NFTL	18
+\\x2dab4ce3490bb50b2ea4c07ab1b6a9cfe29d89b3	COOL20	18
+\\x3b4f7cb9e60362a49dd04eb0091a374d340e3efd	ITAM	18
+\\x3ea6bae5c10ff3f99925bdf29fd3f8bcae5aaa66	SGLD	18
+\\x6c936d4ae98e6d2172db18c16c4b601c99918ee6	LIFE	18
+\\x72377f31e30a405282b522d588aebbea202b4f23	VRN	18
+\\x8d610e20481f4c4f3acb87bba9c46bef7795fdfe	UNT	18
+\\x92d6c1e31e14520e676a687f0a93788b716beff5	DYDX	18
+\\x010a0288af52ed61e32674d82bbc7ddbfa9a1324	OT-aUSDC-30DEC2021	6
+\\x012e0e6342308b247f36ee500ecb14dc77a7a8c1	SKT	8
+\\x24c19f7101c1731b85f1127eaa0407732e36ecdd	SGT	18
+\\x2772b0dd22381932b104d683e73c5ecb5a3a2408	mLOOT	18
+\\x509eece4793b46e4d81c0d4cb0a5a0c26c45e304	FGLD	18
+\\x60f63b76e2fc1649e57a3489162732a90acf59fe	FLURRY	18
+\\x7ae1d57b58fa6411f32948314badd83583ee0e8c	PAPER	18
+\\x85f17cf997934a597031b2e18a9ab6ebd4b9f6a4	NEAR	24
+\\xba7970f10d9f0531941dced1dda7ef3016b24e5b	BGLD	18
+\\xbdb4a359a506b0997a257da767308436c7340245	ACE	18
+\\xcb84d72e61e383767c4dfeb2d8ff7f4fb89abc6e	VEGA	18
+\\xcfbd04b3cef2cf1527f143a49e5dc1e19941d254	LOOTL	18
+\\xf009f5531de69067435e32c4b9d36077f4c4a673	UNV	18
+\\x2610f0bfc21ef389fe4d03cfb7de9ac1e6c99d6e	SKYRIM	18
+\\x32353a6c91143bfd6c7d363b546e62a9a2489a20	AGLD	18
```

Comment:
Could you please remove this line? This is a duplicate entry.

Reply:
Yes sorry, that's right only the last one for `AGLD`. I checked your file locally and the tabs were there.

Reply:
Good eye! Fixed the duplicate record.